data/presets/presets/playground/activitypanel.json

```diff
@@ -0,0 +1,24 @@
+{
+    "icon": "maki-playground",
+    "fields": [
+        "playground/theme",
+        "playground/min_age",
```

Comment:
I think it's okay to add `min_age` and `max_age` to these, but I'm noticing it was a past mistake for these fields to be under the `playground` folder. They can also be used for other features like nightclubs. We can fix this later.